guides/source/routing.md

```diff
@@ -519,7 +519,7 @@ resources :photos do
 end
 ```
 
-You can leave out the `:on` option, this will create the same member route except that the resource id value will be available in `params[:photo_id]` instead of `params[:id]`.
+You can leave out the `:on` option, this will create the same member route except that the resource id value will be available in `params[:photo_id]` instead of `params[:id]`, another difference is that the helper will be `photo_preview_url` and `photo_preview_path`.
```

Comment:
I think ```. Route helpers will also be renamed from `preview_photo_url` and `preview_photo_path` to `photo_preview_url` and `photo_preview_path`.``` would sound better.

Reply:
@gmcgibbon updated. Thanks for your advice.